timelineview/build.gradle

```diff
@@ -1,35 +1,15 @@
-apply plugin: 'com.android.library'
-
-ext {
```

Comment:
This configuration is absolutely necessary for library deployment

Reply:
ok